woocommerce/woocommerce
# Improve visibility of .woocommerce-store-alerts notifications on WooCommerce > Extensions pages

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Initially for review by @woocommerce/alpha-fire before consideration for inclusion in `trunk`. Closes 17944-gh-Automattic/woocommerce.com.

Adjusts CSS so that the appearance of `.woocommerce-store-alerts` when on a WooCommerce > Extensions page does not result in the alert appearing either (a) "hidden" behind the header or (b) rubbing up against the side of the viewport. Screenshot illustrating the problem:

![Screenshot with a "database update" alert appearing behind the header of the Marketplace](https://github.com/woocommerce/woocommerce/assets/53293/f0910fd0-448b-4ac2-bfd0-b552071bcd70)

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch to your environment and compile the WC Admin assets, e.g. with `cd plugins/woocommerce-admin && pnpm i && pnpm start:hot`
2. View a WooCommerce > Extensions page both (a) with and (b) without a notification visible, and confirm that the notification is visible and usable at a variety of screen widths.
    - **Note:** if you need to display a note, you can use the attached patch file ([woocommerce-14-12-32.patch](https://github.com/woocommerce/woocommerce/files/12537175/woocommerce-14-12-32.patch)) to add a notice to 
the site (remember to delete the changes to `class-wc-helper-admin.php` afterwards)
   - **Note 2: Cleanup:** once you've run this, replace the custom method content from the patch file with the following and refresh to remove the note from the DB:
```php
  if ( ! class_exists( 'Automattic\WooCommerce\Admin\Notes\Notes' ) ) {
    return;
  }
  Notes::delete_notes_with_name( self::NOTE_NAME );
```
4. Please test around this change.

### Changelog entry


<details>

#### Significance


#### Type


#### Message 

#### Comment 

Improved visibility/layout of store alerts on WooCommerce > Extensions pages.

</details>
